### DISCUSSION TITLE
Custom Classifiers in Chirpity

### DISCUSSION THREAD

**Discussion by aaruljain** [Mattk70/Chirpity-Electron#124] [Q&A] [answered]
Does Chirpity allow custom classifiers too apart from already existing BirdNET classifiers. I need to do validation for a particular model custom classifier in Mac OS and I can't find BirdNET analyzer GUI or custom classifiers in Chirpity. Please help.

**Comment by Mattk70** [comment, marked as answer]
Hi @aaruljain ,

This question has been discussed here: https://github.com/Mattk70/Chirpity-Electron/issues/48, and explored further here: https://github.com/kahst/BirdNET-Analyzer/discussions/297.

My conclusion was that, while it may be _possible_ to extend Chirpity to take a custom classifier trained using BirdNET-Analyzer, this would not be the way to do it. It is just too complicated.

I may allow custom models to be trained in Chirpity at some point, but this would not help with your apparent use case.

 There is a [Mac installer branch in the BirdNET Analyzer repo](https://github.com/kahst/BirdNET-Analyzer/tree/mac-installer?tab=readme-ov-file). But, it's not had an update for 4 months. Meanwhile, your best bet is to do as I suggest [here](https://github.com/kahst/BirdNET-Analyzer/discussions/338#discussioncomment-9477037)
